r124/outref/ydb312_gtm8182e.txt

```diff
@@ -0,0 +1,56 @@
+# Create the DB
+
+
+# Start INST1 INST2 replication
+==Executing MULTISITE_REPLIC 'START INST1 INST2'==
+Starting Primary Source Server in ##TEST_PATH##
+Starting Passive Source Server and Receiver Server in ##FILTERED##_REMOTE_TEST_PATH_/instance2
+
+# Start INST3 INST4 replication
+==Executing MULTISITE_REPLIC 'START INST3 INST4'==
+Starting Primary Source Server in ##FILTERED##_REMOTE_TEST_PATH_/instance3
+Starting Passive Source Server and Receiver Server in ##FILTERED##_REMOTE_TEST_PATH_/instance4
+
+
+# Running rmv_map script in INST3 to remove instance file mapping
+==Executing MULTISITE_REPLIC 'RUN INST3 ##TEST_COM_PATH##/rmv_map.csh'==
+%GDE-I-LOADGD, Loading Global Directory file 
+	##REMOTE_TEST_PATH##/instance3/mumps.gld
+%GDE-I-VERIFY, Verification OK
+
+%GDE-I-VERIFY, Verification OK
+
+%GDE-I-GDUPDATE, Updating Global Directory file 
+	##REMOTE_TEST_PATH##/instance3/mumps.gld
+==Executing MULTISITE_REPLIC 'RUN INST3 ##SOURCE_PATH##/mumps -run GDE SHOW -INSTANCE  '==
+%GDE-I-LOADGD, Loading Global Directory file 
+	##REMOTE_TEST_PATH##/instance3/mumps.gld
+%GDE-I-VERIFY, Verification OK
+
+%GDE-I-NOACTION, Not updating Global Directory ##REMOTE_TEST_PATH##/instance3/mumps.gld
+
+
+# Copy ydb312gtm8182e.m to INST3
+# Unset [ydb/gtm]_repl_instance env variables  and run ydb312gtm8182e.m to set globals on INST1 and INST3 from INST3
+==Executing MULTISITE_REPLIC 'RUN INST3 unsetenv ydb_repl_instance; unsetenv gtm_repl_instance; $gtm_dist/mumps -run ydb312gtm8182e'==
+    Change to INST1 global directory and set ^a
+    Change to mumps.gld global directory and set ^a
+    (Expecting REPLINSTMISMTCH error)
+%YDB-E-REPLINSTUNDEF, Replication instance environment variable $ydb_repl_instance/$gtm_repl_instance is undefined
+		At M source location ydb312gtm8182e+10^ydb312gtm8182e
+
+YDB>
+# Reset the _repl_instance env vars in INST3
+==Executing MULTISITE_REPLIC 'RUN INST3 setenv gtm_repl_instance "mumps.repl"'==
+==Executing MULTISITE_REPLIC 'RUN INST3 setenv ydb_repl_instance "mumps.repl"'==
+==Executing MULTISITE_REPLIC 'STOP INST1 INST2'==
+Shutting down Passive Source Server and Receiver Server in ##FILTERED##_REMOTE_TEST_PATH_/instance2
+Shutting down Primary Source Server Server in ##TEST_PATH##
+==Executing MULTISITE_REPLIC 'STOP INST3 INST4'==
+Shutting down Passive Source Server and Receiver Server in ##FILTERED##_REMOTE_TEST_PATH_/instance4
+Shutting down Primary Source Server Server in ##FILTERED##_REMOTE_TEST_PATH_/instance3
+# Check and shutdown the DB
+----------------------------------------------------------------------------
+# DB has shutdown gracefully
+spencer:##TEST_PATH##/msr_execute_12.out
```

Comment:
Please remove hardcoded host names from reference files. In fact, it would be better to run check_error_exist.csh on the msr*.out file and avoid the test framework from capturing this expected error at the end of the test.